templates/layout/region--footer.html.twig

```diff
@@ -12,12 +12,39 @@
  * @see template_preprocess_region()
  */
 #}
+{% set classes = [
+  'region',
+  'region-' ~ region|clean_class,
+  'osu-text-bucktoothwhite',
+  'osu-min-h-300',
+  'px-4',
+  'pt-5',
+  'pb-1',
+  'd-flex',
+  'flex-column',
+  'justify-content-between'
+] %}
+
```

Comment:
I added the bootstrap classes on the div here instead of in the footer sass file